src/wallet/wallet.h

```diff
@@ -738,6 +738,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
      * @return next transaction order id
      */
     int64_t IncOrderPosNext(CWalletDB *pwalletdb = NULL);
+    DBErrors ReorderTransactions();
```

Comment:
DBErrors is already exposed in CWallet::LoadWallet I'll probably change that later